packages/frontend-2/components/developer-settings/CreateApplicationSuccessDialog.vue

```diff
@@ -54,6 +50,17 @@ const props = defineProps<{
 
 const isOpen = defineModel<boolean>('open', { required: true })
 
+const {
+  public: { baseUrl }
+} = useRuntimeConfig()
+
+const authUrl = computed(() => {
+  if (props.application?.id) {
+    return `${baseUrl}/authn/verify/${props.application.id}/{code_challenge}`
```

Comment:
What if baseUrl has a trailing slash? Build the URL using `new URL(...).toString()`